core/services/helper/main.py

```diff
@@ -107,6 +107,7 @@ class ServiceMetadata(BaseModel):
     api: str
     sanitized_name: Optional[str]
     works_in_relative_paths: Optional[bool]
+    extras: Optional[Dict[str, str]]
```

Comment:
maybe we should name it `integrations`

Reply:
I think that limits the usage. I don't want to tell people what they can or can't add in these jsons. not sure if that makes sense.

Reply:
I see!